redfish/manager.go

```diff
@@ -217,6 +217,10 @@ type Manager struct {
 	// ManagerNetworkProtocol which represents the network services for this
 	// manager.
 	networkProtocol string
+	// OEMData are all OEM data under top level manager section
+	OemData interface{}
+	// OEMLinks are all OEM data under link section
+	OemLinks interface{}
```

Comment:
Please use common.Links

Reply:
common.Links seems can not pass UT if data like below
```
			"Oem": {
				"Dell": {
					"DellAttributes": [
						{
							"@odata.id": "/redfish/v1/Managers/iDRAC.Embedded.1/Attributes"
						},
						{
							"@odata.id": "/redfish/v1/Managers/System.Embedded.1/Attributes"
						},
						{
							"@odata.id": "/redfish/v1/Managers/LifecycleController.Embedded.1/Attributes"
						}
					],
					"DellAttributes@odata.count": 3,
					"DellTimeService": {
						"@odata.id": "/redfish/v1/Managers/iDRAC.Embedded.1/DellTimeService"
					}
				}
			}
```
Below is UT failure. 
```
manager_test.go:189: Error decoding JSON: json: cannot unmarshal object into Go struct field linkReference.Oem of type common.Links
```